src/treetensornetworks/solvers/tdvp.jl

```diff
@@ -90,42 +112,11 @@ function tdvp(
   insert_function!(sweep_observer!, "sweep_time_printer" => sweep_time_printer)
```

Comment:
Should this get absorbed into the `sweep_printer`?

Reply:
Yes, in principle. Also the current implementation of this is not general enough. One should be able to give a a vector of timesteps (which gets processed together with the other ways of specifying the timestep). We could pass the `region_observer` to the `region_printer`, and similar the `sweep_observer` to the `sweep_printer` in order to access an auxiliary variable like the physical time, computed by some function in the observer. `tdvp` would then have its own default observers and printers (since physical time is not something that matters in most other algorithms).